test/features/environment.py

```diff
@@ -0,0 +1,106 @@
+import os
+from behaving import environment as benv
+
+from behaving.web.steps.browser import named_browser
+
+# Path to the root of the project.
+ROOT_PATH = os.path.realpath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../'))
+
+# Base URL for relative paths resolution.
+BASE_URL = 'http://ckan:3000/'
+
+# URL of remote Chrome instance.
+REMOTE_CHROME_URL = 'http://chrome:4444/wd/hub'
+
+# @see .docker/scripts/init.sh for credentials.
+PERSONAS = {
+    'SysAdmin': dict(
+        name=u'admin',
+        email=u'admin@localhost',
+        password=u'password'
+    ),
+    'Unathenticated': dict(
+        name=u'',
+        email=u'',
+        password=u''
+    ),
+    # This user will not be assigned to any organisations
+    'CKANUser': dict(
+        name=u'ckan_user',
+        email=u'ckan_user@localhost',
+        password=u'password'
+    ),
+    'SalsaAdmin': dict(
```

Comment:
please replace all references to `Salsa` with `Test`

Reply:
Is SalsaAdmin/SalsaEditor an org admin?
If so change it to UserWithOrgAdminGroup or simliar for its persona